### PR TITLE
[enterprise-4.10] Update 4.10 docs per style guide

### PR DESCRIPTION
**Pull request by kelbrown20**
**For version:** 4.10 

**Description:** Updates 4.10 docs with the correct guidelines per updated style guide. Link to style guide: https://redhat-documentation.github.io/supplementary-style-guide/#aarch64 

**Previews:** 

- [Selecting an installation method and preparing a cluster -> Support installation methods for different platforms ](https://kelbrown20.github.io/Docs-previews/previews/Updating-docs-per-architecture-guidelines/installing/installing-preparing.html#supported-installation-methods-for-different-platforms)
- Installing a user-provisioned cluster on bare metal ->
    1.[ Installing RHCOS by using PXE or IPXE booting](https://kelbrown20.github.io/Docs-previews/previews/Updating-docs-per-architecture-guidelines/installing/installing_bare_metal/installing-bare-metal.html#installation-user-infra-machines-pxe_installing-bare-metal) 
    2. [Advanced RHCOS installation reference](https://kelbrown20.github.io/Docs-previews/previews/Updating-docs-per-architecture-guidelines/installing/installing_bare_metal/installing-bare-metal.html#installation-user-infra-machines-advanced_installing-bare-metal) 
- Installing on-premise with Assisted installer
  1. [Installing an on-premise cluster using the Assisted installer](https://kelbrown20.github.io/Docs-previews/previews/Updating-docs-per-architecture-guidelines/installing/installing_on_prem_assisted/installing-on-prem-assisted.html) 
  2. [Installing with the assisted installer -> Setting the cluster details](https://kelbrown20.github.io/Docs-previews/previews/Updating-docs-per-architecture-guidelines/installing/installing_on_prem_assisted/assisted-installer-installing.html#setting-the-cluster-details_assisted-installer-installing) 
- Installing a cluster on AWS in a restricted network with user-provisioned infrastructure 
  1. [Accessing the RHCOS AMI with stream metadata ](https://kelbrown20.github.io/Docs-previews/previews/Updating-docs-per-architecture-guidelines/installing/installing_aws/installing-restricted-networks-aws.html#installation-aws-ami-stream-metadata_installing-restricted-networks-aws)
  2. [RHCOS AMIs for the AWS infrastructure](https://kelbrown20.github.io/Docs-previews/previews/Updating-docs-per-architecture-guidelines/installing/installing_aws/installing-restricted-networks-aws.html#installation-aws-user-infra-rhcos-ami_installing-restricted-networks-aws) 
- Installing a cluster on AWS with customizations 
  1. [Tested instance types for AWS on 64-bit ARM infrastructures](https://kelbrown20.github.io/Docs-previews/previews/Updating-docs-per-architecture-guidelines/installing/installing_aws/installing-aws-customizations.html#installation-aws-arm-tested-machine-types_installing-aws-customizations) 
  2. [Tested instance types for AWS](https://kelbrown20.github.io/Docs-previews/previews/Updating-docs-per-architecture-guidelines/installing/installing_aws/installing-aws-customizations.html#installation-aws-tested-machine-types_installing-aws-customizations)

### CODE REVIEW

File: modules/installation-user-infra-machines-pxe.adoc [hunk 1]
```diff
@@ -232,12 +232,12 @@ This configuration does not enable serial console access on machines with a grap
 ifndef::openshift-origin[]
 [NOTE]
 ====
-To network boot the CoreOS `kernel` on `arm64` architecture, you need to use a version of iPXE build with the `IMAGE_GZIP` option enabled. See link:https://ipxe.org/buildcfg/image_gzip[`IMAGE_GZIP` option in iPXE].
+To network boot the CoreOS `kernel` on 'aarch64' architecture, you need to use a version of iPXE build with the `IMAGE_GZIP` option enabled. See link:https://ipxe.org/buildcfg/image_gzip[`IMAGE_GZIP` option in iPXE].
```
Comment (adellape) [inline modules/installation-user-infra-machines-pxe.adoc:235]:
```suggestion
To network boot the CoreOS `kernel` on `aarch64` architecture, you need to use a version of iPXE build with the `IMAGE_GZIP` option enabled. See link:https://ipxe.org/buildcfg/image_gzip[`IMAGE_GZIP` option in iPXE].
```

**Review by adellape — COMMENTED** [review]
Just 1 comment, but this LGTM based on my understanding of the new SSG guidance.

**Comment by kelbrown20** [comment]
@aleskandro Hey Alessandro! Do you think you could take a look at this PR when you get the chance? Thank you!

**Review by aleskandro — APPROVED** [review]
Thanks @kelbrown20 for the updates! LGTM

**Pull request merged**